stanford_media.module

```diff
@@ -279,17 +280,40 @@ function stanford_media_media_prepare_form(MediaInterface $media, $operation, Fo
   // Display a message to the user to alert them than editing will affect
   // multiple pieces of content.
   if ($count) {
-    $message = \Drupal::translation()->formatPlural(
+    $pluralText = \Drupal::translation()->formatPlural(
```

Comment:
Our current convention is snake case, not camel case.

Reply:
Changed.